src/utils.js

```diff
@@ -64,6 +65,19 @@ export const newClient = () => {
     return new ApifyClient(opts);
 };
 
+/**
+ * Creates an instance of ApifyStorageLocal using options as defined in the environment variables.
+ * @return {ApifyStorageLocal}
+ */
+const newApifyStorageLocal = () => {
+    const localStorageDir = process.env[ENV_VARS.LOCAL_STORAGE_DIR] || LOCAL_ENV_VARS[ENV_VARS.LOCAL_STORAGE_DIR];
+    const opts = {
+        dbDirectoryPath: localStorageDir,
```

Comment:
`dbDirectoryPath` points to the directory containing the SQL lite file? Why not simply call it `directoryPath` or `storageDir` ? The fact that it's a "db" is an implementation detail.

Reply:
Good point. Will change.